src/collective/fingerpointing/configure.zcml

```diff
@@ -1,11 +1,17 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
     xmlns:five="http://namespaces.zope.org/five">
 
   <five:registerPackage package="." />
 
   <include package="Products.CMFCore" file="permissions.zcml" />
 
+  <permission
+      id="collective.fingerpointing.showlog"
+      title="collective.fingerpointing: Show the Log"
```

Comment:
not at all - i even asked @pcdummy to rename this in another comment but he did not yet find the time to to so. ;-)